### PR TITLE
do not check mounts entering windows container studio

### PR DESCRIPTION
**Pull request by mwrock**
Windows containers do not use shared drives for mounting local volumes so the `check_mounts` function provides no value when run in Windows container mode. Also, this incurs a `docker run` which takes much longer on Windows 10 hosts for windows containers (like ~ 20 seconds) so removing this check significantly speeds up the studio startup time.

Signed-off-by: mwrock <matt@mattwrock.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/hab/src/command/studio/docker.rs [hunk 1]
```diff
@@ -197,9 +201,7 @@ where
         cmd_args.push(vol.as_ref().into());
     }
 
-    if !is_serving_windows_containers(docker_cmd) {
```
Comment (raskchanky) [inline components/hab/src/command/studio/docker.rs]: Was the removal of this check intentional?  Is this related to mounts too?

Comment (mwrock) [reply components/hab/src/command/studio/docker.rs]: since this PR would not enter this function when using windows containers, this check would always be `false`

**Comment by mwrock** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/346447758).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/346447758).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265
